Documentation/upgrades/upgrade_3_3.md

```diff
@@ -8,7 +8,73 @@ Before [starting an upgrade](#upgrade-procedure), read through the rest of this
 
 ### Client upgrade checklists
 
-3.3 introduces breaking changes (TODO: update this before 3.3 release).
+3.3 introduces breaking changes.
+
+`etcdserver.EtcdServer` has changed the type of its member field `*etcdserver.ServerConfig` to `etcdserver.ServerConfig`. And `etcdserver.NewServer` now takes `etcdserver.ServerConfig`, instead of `*etcdserver.ServerConfig`.
```

Comment:
is `etcdserver.ServerConfig` user facing?

Reply:
Yeah kubernetes is using it for tests.